lib/policy/expressions/environment.go

```diff
@@ -19,6 +19,7 @@ func BotEnvironment() (*cel.Env, error) {
 	return New(
 		// Variables exposed to CEL programs:
 		cel.Variable("remoteAddress", cel.StringType),
+		cel.Variable("contentLength", cel.IntType),
```

Comment:
I checked, [IntType is int64](https://pkg.go.dev/github.com/google/cel-go@v0.26.1/common/types#Int).